pyrokinetics/gk_code/GKOutputReader.py

```diff
@@ -223,11 +256,18 @@ def _set_eigenvalues(
         data["growth_rate"] = (("kx", "ky", "time"), growth_rate.data)
         data["mode_frequency"] = (("kx", "ky", "time"), mode_frequency)
         data["eigenvalues"] = (("kx", "ky", "time"), eigenvalue)
+
+        eigenvalue_units = eigenvalues_units(data.local_norm.pyrokinetics)
+
+        data = data.pint.quantify(eigenvalue_units)
```

Comment:
This is a little confusing reusing the function name, maybe just do:

```suggestion
        data = data.pint.quantify(eigenvalues_units(data.local_norm.pyrokinetics))
```